www/include/configuration/configServers/listServers.ihtml

```diff
@@ -129,3 +129,9 @@
 
 </script>
 {/literal}
+
+{literal}
+<script>
+    formatDateMoment();
+</script>
+{/literal}
```

Comment:
It is not possible to call the function directly after the bracket? This will avoid closing the "script" and "literal" tags to open them again afterwards.